### PR TITLE
Show JS version and custom UIs in dev info

### PR DESCRIPTION
**Pull request by c727**
- add frontend JS version to dev info
- add custom UI info to dev info

A problem with custom UIs is that users often run old, cached, versions. This shows the ~~installed~~ running version now.

Andrey already has this for his custom-ui. This should provide an interface for everyone

code to include in your custom-state-card:
```javascript
const CUSTOM_UI_NAME = 'state-card-tiles';
const CUSTOM_UI_VERSION = '20180308';
const CUSTOM_UI_URL = 'https://github.com/c727/home-assistant-tiles';

if (!window.CUSTOM_UI_LIST) {
  window.CUSTOM_UI_LIST = [];
}
window.CUSTOM_UI_LIST.push({
  name: CUSTOM_UI_NAME,
  version: CUSTOM_UI_VERSION,
  url: CUSTOM_UI_URL
}); 
```
cui2

### CODE REVIEW

**Review by andrey-git — APPROVED** [review]
Looks good to me, but let's wait for @balloob

**Comment by balloob** [comment]
Yeah I'm 👍  on this. Just not sure what's with the top "Custom UI's" in the screenshot? I don't see it in the code either? (I also think it shouldn't be there, just 1 place to render this data seems fine)

File: panels/dev-info/ha-panel-dev-info.html [hunk 1]
```diff
@@ -123,6 +123,19 @@
           <p>
             Path to configuration.yaml: [[hass.config.core.config_dir]]
           </p>
+          <p>
```
Comment (balloob) [inline panels/dev-info/ha-panel-dev-info.html]: Can we move this under the version of Python/ Polymer ?

**Comment by c727** [comment]
@balloob "Custom UI 20180216" is added by Andrey's state-card where I got the inspiration for this - It's not included in this PR

**Review by balloob — APPROVED** [review]

**Pull request merged**